src/test/java/project/web/GasStationControllerTest.java

```diff
@@ -75,10 +85,12 @@ void getGasPriceInfoTestWithInvalidInput() {
         when(gasStationService.getGasPriceInfo(fuelType)).thenReturn(null);
 
         // WHEN
-        ResponseEntity<GasPriceInfoDTO> response = gasStationController.getGasPriceInfo(fuelType);
+        ResponseEntity<GasPriceInfoDTO> gasPriceInfo = gasStationController.getGasPriceInfo(fuelType);
```

Comment:
here the naming is much better

Reply:
here i remove Info and leave it as gasPrice.